docs/UPGRADE.md

```diff
@@ -158,12 +158,11 @@ web-based script. To do so:
 
 ### Update Javascript libraries and build.js
 
-The official .tar.gz releases, and the stable branches in git (e.g.
-`ojs-stable-3_1_1`), contain precompiled javascript. If you are installing
-OJS using either of those and have not modified your Javascript, you do not
-need to compile Javascript.
+The official .tar.gz releases contain precompiled javascript. If you are installing
+OJS using this method, you do not need to compile Javascript.
 
-If you are using the git `master` branch, or have made changes to your
+If you are using the git `main` branch, or any of the stable branches (e.g.
+`ojs-stable-3_3_0`), or have made changes to your
```

Comment:
This should be `stable-3_3_0`, not `ojs-stable-3_3_0`, I think.

Reply:
Updated to `stable-3_3_0`